.devcontainer/docker-compose.yml

```diff
@@ -21,17 +21,16 @@ services:
     # (Adding the "ports" property to this file will not forward from a Codespace.)
 
   db:
-    image: postgis/postgis:12-3.0-alpine
+    image: postgis/postgis:14-3.3-alpine
```

Comment:
We should really get everything using this image - 15 can't be installed on ubuntu via apt yet (for pg_dump etc), so 14 is most convenient

Reply:
good call, `pg_dump` is needed for some builds and may come in very handy for future data warehouse stuff. noted this in the "standardize environment" issue